plugins/inputs/gnmi/sample.conf.in

```diff
@@ -0,0 +1,119 @@
+# gNMI telemetry input plugin
```

Comment:
What is this file for?

Reply:
This is the file used to generate `sample.conf` by embedding `plugins/common/tls/client.conf`... :-)